software-full.json

```diff
@@ -31816,258 +31861,40 @@
             ],
             "archived": false,
             "versions": {
-                "3.3.1": [
-                    "CVE-2024-5535"
```

Comment:
There seem to be an issue with CVE matching, a lot of CVE:s have been removed in this update.
we need to check why.